Automattic/jetpack
# Protect: add admin notice when vulnerabilities detected

This PR shows and admin notice (warning) when Protect plugin detects vulnerabilities.

Fixes #

#### Changes proposed in this Pull Request:

* Protect: add admin notice when vulnerabilities detected

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Since nums of vulnerabilities are hardcoded, a way to show the warning is hardcoding the [has_vulnerabilities()](https://github.com/Automattic/jetpack/blob/9a6123a8ef24fd801d7a949fcce88baa687d86ea/projects/plugins/protect/src/class-status.php#L87) function in your testing dev-env, returning `true`:

```php
public static function has_vulnerabilities() {
	return true;
	$status = self::get_status();
	return isset( $status->num_vulnerabilities ) && is_int( $status->num_vulnerabilities ) && 0 < $status->num_vulnerabilities;
}
```
* Go to wp-admin page, for instance, posts:
* Confirm you see the warning there
* Confirm the link leads to the Protect overview page
<img width="892" alt="image" src="https://user-images.githubusercontent.com/77539/162482959-cb93b0e6-33ae-4bc8-959c-f49341ff945f.png">
